native/cocos/2d/renderer/Batcher2d.cpp

```diff
@@ -0,0 +1,258 @@
+#include <2d/renderer/Batcher2d.h>
+#include <cocos/base/TypeDef.h>
+#include <cocos/core/Root.h>
+#include <renderer/pipeline/Define.h>
+#include <scene/Pass.h>
+#include <deque>
```

Comment:
I think should add ccstd::deque as other container.

Reply:
done.remove it